Source/WebKit/UIProcess/WebProcessPool.h

```diff
@@ -522,7 +522,10 @@ class WebProcessPool final
 
 #if USE(ATSPI)
     const String& accessibilityBusAddress() const;
+    const String& accessibilityBusName() const;
     const String& sandboxedAccessibilityBusAddress() const;
+
+    const String& nextAccessibilityBusName();
```

Comment:
I would name this `generateNextAccessibilityBusName`.

Reply:
Done